app/depict/src/main/java/org/openscience/cdk/depict/FreeHepWrapper.java

```diff
@@ -98,13 +116,55 @@ public void dispose() {
         g2.dispose();
     }
 
+    // Documents cited below:
+    // [PLDS92] PostScript Language Document Structuring Conventions Specification, Version 3.0, 25 September 1992
+    // [EGFF96] J.D.Murray & W. vanPyper, Encyclopedia of Graphics File Formats 2nd ed., O'Reilly & Assoc., 1996
+
     @Override
     public String toString() {
         String result = new String(bout.toByteArray(), StandardCharsets.UTF_8);
         // we want SVG in mm not pixels!
         if (fmt.equals(Depiction.SVG_FMT)) {
             result = result.replaceAll("\"([-+0-9.]+)px\"", "\"$1mm\"");
         }
+        if (fmt.equals(Depiction.EPS_FMT)) {
```

Comment:
The performance difference is probably minimal, but maybe use an 'else-if' here?